src/convex/ConvexBoosterController.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import {IController} from "../core/IController.sol";
+
+interface IBooster {
+    function poolInfo(uint256) external view returns(address, address, address, bool, address);
+}
+
+contract ConvexBoosterController is IController {
+
+    address public immutable BOOSTER;
+
+    /// @notice deposit(uint256,uint256)
+    bytes4 constant DEPOSIT = 0xe2bbb158;
```

Comment:
convex booster also exposes a depositAll(uint) function

Reply:
Need this for partial deposits from the UI, can't always force users to deposit all tokens. Also `depositAll(uint)` just calls `deposit(uint,uint)` internally so it seemed redundant.

Reply:
Having both functions just helps us cover all vectors of interactions with the contract. Client can choose to do what it wants.

Reply:
I think it's better to shift the logic to the caller instead of adding redundancies in the controller itself. These should be as lightweight as possible.